### DISCUSSION TITLE
Don't use the full path in the media_file table

### DISCUSSION THREAD

**Discussion by JulianWgs** [navidrome/navidrome#3121] [Ideas]
When migrating from one to another server you lose of all of your listens if you either
- change the music folder path or
- dont use [this Python tool](https://github.com/kgarner7/navidrome-migrate).

I think this is easily fixable if the table would only use the relative path within the music folder. I am not a go developer, but I with some guidance I could take on this issue.

PS: Thanks for the awesome software!

**Comment by deluan** [comment]
Hey, thanks for trying Navidrome! This is being worked on #2709, you will even be able to move files around, restructure your folders and Navidrome will figure out any moves.

**Comment by JulianWgs** [comment]
Thank you for the quick answer! I really searched thoroughly, but I missed that PR...
